provider/cluster_rosa_classic_state.go

```diff
@@ -51,21 +51,16 @@ type ClusterRosaClassicState struct {
 }
 
 type Sts struct {
-	OIDCEndpointURL  types.String      `tfsdk:"oidc_endpoint_url"`
-	Thumbprint       types.String      `tfsdk:"thumbprint"`
-	RoleARN          types.String      `tfsdk:"role_arn"`
-	SupportRoleArn   types.String      `tfsdk:"support_role_arn"`
-	InstanceIAMRoles InstanceIAMRole   `tfsdk:"instance_iam_roles"`
-	OperatorIAMRoles []OperatorIAMRole `tfsdk:"operator_iam_roles"`
+	OIDCEndpointURL    types.String    `tfsdk:"oidc_endpoint_url"`
+	Thumbprint         types.String    `tfsdk:"thumbprint"`
+	RoleARN            types.String    `tfsdk:"role_arn"`
+	SupportRoleArn     types.String    `tfsdk:"support_role_arn"`
+	InstanceIAMRoles   InstanceIAMRole `tfsdk:"instance_iam_roles"`
+	AccountRolePrefix  types.String    `tfsdk:"account_role_prefix"`
```

Comment:
Not sure if this should be passed of figured out from one of the acc roles

Reply:
it's the same as rosa cli determine the policies names - it's getting the account roles themselves

Reply:
You're right - I only need it in the rosa operator data source and I changed it to be optional with a default value.